api-demo/src/start.ts

```diff
@@ -0,0 +1,72 @@
+import * as os from 'os';
+import * as colors from 'colors';
+import { TSDocParser, ParserContext, DocComment } from '@microsoft/tsdoc';
+import { Formatter } from './Formatter';
+
+console.log(colors.cyan('*** TSDoc API demo ***'));
+console.log();
+
+const tsdocParser: TSDocParser = new TSDocParser();
+
+const inputBuffer: string = [
+  /* line  1 */ ' ',
+  /* line  2 */ '  /**',
+  /* line  3 */ '   * Adds two numbers together.',
+  /* line  4 */ '   *',
+  /* line  5 */ '   * @remarks',
+  /* line  6 */ '   * This method is part of the {@link core-libary/Math | Math subsystem}.',
+  /* line  7 */ '   * This HTML tag has an error: <tag',
+  /* line  7 */ '   *',
+  /* line  8 */ '   * @param x - The first number to add',
+  /* line  9 */ '   * @param y - The second number to add',
+  /* line  0 */ '   * @returns The sum of `x` and `y`',
+  /* line 11 */ '   *',
+  /* line 12 */ '   * @beta',
+  /* line 13 */ '   */',
+  /* line 14 */ '            '
+].join('\n');
```

Comment:
This should be in a real source file.

Reply:
Maybe put it in a folder parallel to `src` called `inputs` or something.

---
In reply to: [210776656](https://github.com/Microsoft/tsdoc/pull/46#discussion_r210776656) [](ancestors = 210776656)